### ISSUE TITLE
setup.py installs meta-files to the wrong directory

### ISSUE THREAD

**Issue by lieryan** [ixmatus/inflector#4]
Inflector does not install correctly, it leads to a faulty installation as such:

```
/virtualenv/CHANGES
/virtualenv/DEBT
/virtualenv/LICENSE
/virtualenv/README.md
/virtualenv/lib/python2.7/site-packages/Inflector-2.0.7-py2.7.egg-info
/virtualenv/lib/python2.7/site-packages/inflector/__init__.py
```

Installed packages should not put its files to the root of the environment.

**Comment by yosida95** [comment]
This problem still exists.  The cause of this is here: https://github.com/ixmatus/inflector/blob/78739f8cb9df3e126bf13f6baa07b958651c4f75/setup.py#L24

data_files will be installed to relative location from sys.prefix, so README.md, LICENSE, and VERSION will be put to sys.prefix. For instance,  these files are put to "/usr/{README.md,LICENSE,VERSION}" if this module has been installed to system python (/usr/bin/python).

**Comment by ixmatus** [comment]
I don't even use Python professionally anymore. I'm happy to accept
pull-requests or approve a full-time maintainer.
ᐧ

On Wed, Apr 22, 2015 at 3:42 AM, Kohei YOSHIDA <notifications@github.com>
wrote:

> This problem still exists. The cause of this is here:
> https://github.com/ixmatus/inflector/blob/78739f8cb9df3e126bf13f6baa07b958651c4f75/setup.py#L24
>
> data_files will be installed to relative location from sys.prefix, so
> README.md, LICENSE, and VERSION will be put to sys.prefix. For instance,
> these files are put to "/usr/{README.md,LICENSE,VERSION}" if this module
> has been installed to system python (/usr/bin/python).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ixmatus/inflector/issues/4#issuecomment-95079060>.
>



-- 
Parnell Springmeyer
parnell@digitalmentat.com | digitalmentat.com | 0xDCCF89258EAD874A
<http://pgp.mit.edu/pks/lookup?op=get&search=0xDCCF89258EAD874A>

**Comment by EmilStenstrom** [comment]
It seems it's not possible to uninstall Inflector any longer. It complains that I haven't given it permission to delete /usr/local/LICENSE which I assume that this is of the same reason as this issue solves.

I think this PR makes sense, needs another release to pypi too.

**Comment by ixmatus** [comment]
I merged the PR that fixes this issue and I also released `2.0.12` to PyPi. @EmilStenstrom thanks for the reminder, sorry for the lack of attention :)

**Comment by EmilStenstrom** [comment]
@ixmatus: it’s we that should be thankful for you putting your free time into building open source software for everyone else. Each commit is a new gift for us all, so don’t feel bad for not giving gifts. Thank you!
